packages/react/src/components/Accordion/Accordion.tsx

```diff
@@ -38,7 +38,7 @@ export interface AccordionProps extends UIComponentProps, ChildrenComponentProps
   /** Initial activeIndex value. */
   defaultActiveIndex?: number[] | number
 
-  /** Only allow one panel open at a time. */
+  /** Only allow one panel to be expanded at a time. */
```

Comment:
Consistency: Accordion uses the word `expanded` to describe its panels' open states, and that term is also used in describing other props.